task-1/question-1.js

```diff
@@ -0,0 +1,17 @@
+//question: 
+// 1)Write a   program which reads a string from standard input reverses the string and writes it to the standard output.
+// a)The program should run in a   
+// stand-by mode and should not be terminated after reversing the first string.
+
+
+// solution:
+var stdin = process.stdin;
```

Comment:
You can probably use `const` instead of `var`

Reply:
done